api/routes/auth.route.js

```diff
@@ -2,11 +2,12 @@ import express from "express";
 import {login, logout, register} from "../controllers/auth.controller.js";
 import {withValidationHandler} from "./validation/withValidationHandler.js";
 import {loginValidation, registerValidation} from "./validation/auth.validation.js";
+import { globalErrorCatcher } from "../middleware/globalErrorCatcher.js";
 
 const router = express.Router();
 
 router.post("/register", ...registerValidation, withValidationHandler(register))
 router.post("/login", ...loginValidation, withValidationHandler(login))
-router.post("/logout", logout)
+router.post("/logout", globalErrorCatcher(logout))
```

Comment:
dont u want to wrap all of them with ur globalErrorCatcher(..) ... did u trying to c if a middleware could be used?

Reply:
Yeah I tried but nothing worked only this is working